app/views/videos/index.html.erb

```diff
@@ -5,6 +5,10 @@
 
 <div class='container'>
   <div class="row">
+    <% flash.each do |key, msg| %>
+      <div class="alert alert-<%= key %>">
+        <%= msg %>
```

Comment:
I saw you were using bootstrap and tried to leverage their alert styles. See http://getbootstrap.com/components/#alerts